MOOSE Docs Slides (#252)
shannon-lab/zapdos: doc/workshops/tutorial/content/examples/tutorial3-fenix.md

> # Tutorial 3 - Kinetic Implementation

Reviewer: ```suggestion
# Tutorial 3 - Kinetic Implementation using FENIX
```

!media media/FENIX_logo_blue.png

>        style=width:50%;display:block;margin-top:1.5em;margin-left:auto;margin-right:auto;box-shadow:none;

Reviewer: This is so big that on my ultrawide display, the link doesn't even show up. I would drop the size of this.

Author: I have adjusted this to 30% let me know if that is a good size.

- [Ray Tracing Module](https://mooseframework.inl.gov/modules/ray_tracing/) (Kinetic Plasma Foundation)
- [Heat Transfer](https://mooseframework.inl.gov/modules/heat_transfer/index.html)
- [Solid Mechanics](https://mooseframework.inl.gov/modules/solid_mechanics/index.html)
- [Theraml Hydraulics](https://mooseframework.inl.gov/modules/thermal_hydraulics/index.html)

!---

# style=margin-top:1em;

# PIC in FENIX

In [!cite](lieberman1994principles) this tutorial problem is solved with a PIC code. A core part of the FENIX framework is the PIC capabilities that are being developed. As a very basic benchmarking problem the PIC example presented in [!cite](lieberman1994principles) has been replicated in FENIX

- 100 Macroparticles are placed on a 100 element mesh
- Each particle is initially placed exactly in the center of each cell
- Leap Frog particle stepping is used
- $\Delta t$ = $1\times 10^{-9} [\text{s}]$
- Total simulation time of $2.5\times 10^{-8} [\text{s}]$
- The potential is represented with first order nodal basis functions

# style=margin-top:1em;

!bibtex bibliography

!---

# Results Comparison

> All figures show FENIX results super imposed on figures 2.2.a, 2.2.b, and 2.2.c from [!cite](lieberman1994principles). Results from [!cite](lieberman1994principles) are in black and grey while FENIX results are in color.

Reviewer: ```suggestion
All figures show FENIX results superimposed on figures 2.2.a, 2.2.b, and 2.2.c from [!cite](lieberman1994principles). Results from [!cite](lieberman1994principles) are in black and grey while FENIX results are in color.
```

!row!

!media media/lieberman-particles.png
       style=width:93%;display:block;margin-top:0.5em;margin-left:auto;margin-right:auto;
       caption=Particle Population Evolution

!media media/lieberman-potential.png
       style=width:100%;display:block;margin-top:0.5em;margin-left:auto;margin-right:auto;
       caption=Electrostatic Potential Evolution

!media media/lieberman-vdf.png
       style=width:100%;display:block;margin-top:0.5em;margin-left:auto;margin-right:auto;
       caption=Particle Phase Space Evolution
!row-end!

!---

# Current PIC Status

- Leap frog and Boris particle stepping have been implemented and verified.
- Electrostatic capabilities have been verified for all first order finite elements in libMesh.
- Uniform random particle initialization for all first order finite elements in libMesh has been verified.

  - Bounding box particle initialization has also been verified.

> - Particle based current density source calculations is near completion and will be verified.

Reviewer: I would  re-organize a bit around "verified" and "in-development" activities since there is a lot of word repetition here ("has been verified").

Author: I separated this into two subsections with lists of what is done and what is under work.

- Particle collision capabilities are the next to be developed.

!---

# Future PIC Work

- Continued verification efforts by replicating Kinetic instabilities

  - Landau Damping
  - Two Stream
  - Dory-Ghast-Haris Instability

- Replicating more complex analytic plasma solutions
- Benchmarking against other codes

> - Validation by replicating experimental data

Reviewer: ```suggestion
- Validation by replicating experimental conditions and data
```
